sdk/src/main/scala/com/horizen/backup/BoxIterator.java

```diff
@@ -54,6 +56,22 @@ public Optional<BackupBox> nextBox() throws RuntimeException {
         return nextBox(false);
     }
 
+    public List<Box<Proposition>> getNextBoxes(int nElement, Optional<byte []> keyToSeek) {
```

Comment:
Let's add the UTs for this method with the real storage usage, where we will put real boxes (both coin and non-coin) and also some trash-records (not a boxes).